docs/infrastructure_and_maintenance/cache/persistence_cache.md

```diff
@@ -40,7 +40,7 @@ For further details on which calls are cached or not, see details in the [Symfon
 which has info on cache use in two places:
 
 - Symfony Cache tab: for Symfony Cache itself, the tab shows cache lookups to cache backends
-- eZ Platform tab: shows calls made to database back end, and if they are cached or not
+- Ibexa tab: shows calls made to database back end, and if they are cached or not
```

Comment:
```suggestion
- [[= product_name_base =]] tab: shows calls made to database back end, and if they are cached or not
```
I think vale is right here - if we rename the company we will probably rename this tab as well 😅